server/package.json

```diff
@@ -0,0 +1,15 @@
+{
+  "name": "server",
+  "version": "1.0.0",
+  "description": "",
```

Comment:
lets add a small description for our server

Reply:
done